modules/tensor_mechanics/tests/crystal_plasticity/crysp.i

```diff
@@ -54,9 +54,7 @@
 
 [Kernels]
   [./TensorMechanics]
-    disp_z = uz
-    disp_y = uy
-    disp_x = ux
+    displacements = 'ux uy uz'
```

Comment:
That is definitely the goal--it would reduce the input files even further!  But some cases do not need/ want mesh displacement, so setting ```displacements ``` in the ```[GlobalParams]``` block causes some input files to exodiff fail.